src/planner/expand_hypertable.c

```diff
@@ -1046,11 +1046,16 @@ ts_plan_expand_hypertable_chunks(Hypertable *ht, PlannerInfo *root, RelOptInfo *
 	/* Can have zero chunks. */
 	Assert(num_chunks == 0 || chunks != NULL);
 
+	PlanRowMark *oldrc = get_plan_rowmark(root->rowMarks, rti);
+	if (oldrc)
+	{
+		oldrc->isParent = true;
+	}
```

Comment:
Let's add a comment about why this is necessary.

Reply:
done